content/en/docs/gke/deploy/reasons.md

```diff
@@ -7,10 +7,9 @@ weight = 10
 Running Kubeflow on GCP brings you the following features:
 
   * You use
-    [Deployment Manager](https://cloud.google.com/deployment-manager/docs/) to
+    [Cloud Native Resource Manager](https://cloud.google.com/config-connector/docs) to
     declaratively manage all non-Kubernetes resources (including the GKE 
-    cluster). Deployment Manager is easy to customize for your particular use
-    case.
+    cluster).
   * You can take advantage of 
```

Comment:
Suggestion:

- Replace "autoscaling to scale" and the docs link (GKE -> GKE Cluster Autoscaler) with a more precise definition of this GCP feature:

```diff
- * You can take advantage of 
+ * You can use GKE's [Cluster Autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler)
+ to automatically resize the number of nodes in a node pool in your cluster depending on the workload demands
- your cluster horizontally